backend/src/main/java/com/soen341/instagram/service/impl/PictureService.java

```diff
@@ -22,12 +28,19 @@
 import java.nio.file.Paths;
 import java.util.Date;
 import java.util.Optional;
+import java.util.Set;
 import java.util.List;
 import java.util.stream.Collectors;
 
 @Service("pictureService")
 public class PictureService {
 
+	@Autowired
+	private AccountRepository accountRepository;
+	@Autowired
+	@Qualifier("UserDetailsService")
```

Comment:
same comment as the one for the `comment` class. `UserDetailsService` is never used so remove it.